liquidprompt

```diff
@@ -2039,6 +2051,34 @@ _lp_node_env_color() {
     lp_node_env_color="${LP_COLOR_NODE_VENV}${lp_node_env}${NO_COL}"
 }
 
+_lp_perl_env() {
+    (( LP_ENABLE_PERL_VENV )) || return 2
+
+    local ret
+
+    if [[ "$_LP_PERL_VENV_PROGRAM" = "perlbrew" ]] ; then
+        local perlbrew_perl_ver="$(perlbrew use)"
+        
+        # Example: Currently using perl-5.38.0
+        __lp_escape "${perlbrew_perl_ver##*perl-}"
+        lp_perl_env=$ret
+    elif [[ "$_LP_PERL_VENV_PROGRAM" = "plenv" ]] ; then
+        local plenv_ver="$(plenv version)"
+        # Only first word is necessary as for rbenv
```

Comment:
This comment mentions `rbenv`. A typo? Maybe `plenv` works exactly the same way as `rbenv`?

Reply:
Yes, as you can read from the plenv documentation it is based on rbenv

> Most of part was inspired from [rbenv](https://github.com/sstephenson/rbenv).

And from the license section:

> plenv uses rbenv code